src/Tests/Microsoft.TemplateEngine.Cli.UnitTests/Microsoft.TemplateEngine.Cli.UnitTests.csproj

```diff
@@ -8,12 +8,20 @@
     <IsUnitTestProject>true</IsUnitTestProject>
     <EnableStyleCopAnalyzer>true</EnableStyleCopAnalyzer>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
+
+    <!-- NU1608: Disable dependency constraints. Make packages referenced by the project directly rather than packages from other sources. -->
+    <!-- NETSDK1023: Disable implicit reference so that packages can be referenced in the project. -->
+    <NoWarn>$(NoWarn);NU1608;NETSDK1023</NoWarn>
```

Comment:
I assume NU1608 is needed because this is overriding the version that's in arcade, correct?  Why is 1023 needed those? I'm surprised that adding this reference is triggering that since that should only trigger for packages that come with the runtime

Reply:
It looks like the arcade version is covered, maybe we should just have to wait for the arcade update. Aslo, I'm confused why the error information is different between 7.0.4xx and 8.0.1xx. Is it due to CPM?